### PR TITLE
docs: update vault helm doc

### PR DESCRIPTION
**Pull request by jasonodonnell**
Updated Vault Helm documentation for new version of the chart and the Docker image.

### CODE REVIEW

File: website/source/docs/platform/k8s/helm.html.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ and consider if they're appropriate for your deployment.
 
   * `image` (`string: "vault:latest"`) - The name of the Docker image (including any tag) for the containers running Vault. **This should be pinned to a specific version when running in production.** Otherwise, other changes to the chart may inadvertently upgrade your Vault version.
 
+  * `tlsDisable` (`boolean: true`) - When set to `true`, changes URLs from `https` to `http` (such was the `VAULT_ADDR` environment variable on the Vault pods).
```
Comment (catsby) [inline website/source/docs/platform/k8s/helm.html.md]: > such was the `VAULT_ADDR` environment variable on the Vault pods

Such was it what? It's not clear what `tlsDisable` is doing related to `VAULT_ADDR`

File: website/source/docs/platform/k8s/helm.html.md [hunk 2]
```diff
@@ -99,19 +101,44 @@ and consider if they're appropriate for your deployment.
...
       Type of the volume, must be one of "configMap" or "secret". Case sensitive.
 
       - `name` (`string: required`) -
       Name of the configMap or secret to be mounted. This also controls the path
-      that it is mounted to. The volume will be mounted to `/vault/userconfig/<name>`.
+      that it is mounted to. The volume will be mounted to `/vault/userconfig/<name>` by default
+      unless `path` is configured.
+      
+      - `path` (`string: /vault/userconfigs`) -
+      Name of the path where a configMap or secret are mounted.  If none specified
```
Comment (catsby) [inline website/source/docs/platform/k8s/helm.html.md]: > configMap or secret are mounted

configMap or secret _is_ mounted (?)

Comment (catsby) [reply website/source/docs/platform/k8s/helm.html.md]: Also `If none specified` -> `If not specified`

File: website/source/docs/platform/k8s/helm.html.md (hunk 1 above)
Comment (jasonodonnell) [reply website/source/docs/platform/k8s/helm.html.md]: Good catch, clarified!

File: website/source/docs/platform/k8s/helm.html.md (hunk 2 above)
Comment (jasonodonnell) [reply website/source/docs/platform/k8s/helm.html.md]: Fixed 👍

**Pull request merged**